src/Data/MongoDBMock/doggos.js

```diff
@@ -0,0 +1,89 @@
+const doggos = [
+  {
+    dogId: 'VIII-27165',
+    dogName: 'Nosek',
+    kennelName: 'Narnia Baltica',
+    pkr: 'VIII-27165',
+    registrationNumber: '5491/XLII/11',
+    breed: 'Golden Retriever',
+    dateOfBirth: new Date(2011, 4, 22),
+    color: 'gold',
+    chipOrTattoo: 616093900044541,
+    sex: 'pies',
+    owner: 'Jan Kowalski',
```

Comment:
This should be connected to user id if possible

Reply:
no, really I dont want to connect that. Owner is totally not important person, it is needed just to kennel klub documentation. We don't use it anywhere in our application (it will be used just when generate a documents to print)